vars/infra.groovy

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env groovy
 
+Boolean isRunsOnJenkinsAzure() {
```

Comment:
I like a different shade of red better: `isRunningOnJenkinsInfra` both for the nicer name in Java, still being a nice Groovy property (`runningOnJenkinsInfra`) and making less unnecessary assumptions over where the infra actually is hosted.